scripts/site/generate-site.js

```diff
@@ -142,10 +166,15 @@ function generate(target) {
           fs.readFileSync(path.join(componentDirPath, 'doc/index.en-US.md')),
           `components/${componentName}/doc/index.en-US.md`
         ),
+        standalone: !fs.existsSync(path.join(componentDirPath, 'demo/module')),
```

Comment:
Determine whether the component has been migrated into standalone mode by the existence of `component/xxx/demo/module` file.